clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -111,10 +112,34 @@ interface NexusIqService {
         val packageUrl: String
     )
 
+    data class Organizations(
+        val organizations: List<Organization>
+    )
+
+    data class Organization(
+        val id: String,
+        val name: String,
+        val tags: List<Tag>
+    )
+
+    data class Tag(
+        val id: String,
+        val name: String,
+        val description: String,
+        val color: String
+    )
+
     /**
      * Retrieve the details for multiple [components].
      * See https://help.sonatype.com/iqserver/automating/rest-apis/component-details-rest-api---v2.
      */
     @POST("api/v2/components/details")
     suspend fun getComponentDetails(@Body components: ComponentsWrapper): ComponentDetailsWrapper
+
+    /**
+     * Produce a list of all of organizations and associated information that are viewable for the current user.
```

Comment:
Nit: I think there is one superfluous "of" in the comment. "Produce a list of all organizations ..."